src/trello.coffee

```diff
@@ -1,112 +1,202 @@
+  true
+
+##############################
+# API Methods
+##############################
+
+createCard = (msg, list_name, cardName) ->
+  msg.reply "Sure thing boss. I'll create that card for you."
+  ensureConfig msg.send
+  id = lists[list_name.toLowerCase()].id
+  trello.post "/1/cards", {name: cardName, idList: id}, (err, data) ->
+    msg.reply "There was an error creating the card" if err
+    msg.reply "OK, I created that card for you. You can see it here: #{data.url}" unless err
+
+showCards = (msg, list_name) ->
+  msg.reply "Looking up the cards for #{list_name}, one sec."
+  ensureConfig msg.send
+  id = lists[list_name.toLowerCase()].id
+  msg.send "I couldn't find a list named: #{list_name}." unless id
+  if id
+    trello.get "/1/lists/#{id}", {cards: "open"}, (err, data) ->
+      msg.reply "There was an error showing the list." if err
+      msg.reply "Here are all the cards in #{data.name}:" unless err and data.cards.length == 0
+      msg.send "* [#{card.shortLink}] #{card.name} - #{card.shortUrl}" for card in data.cards unless err and data.cards.length == 0
+      msg.reply "No cards are currently in the #{data.name} list." if data.cards.length == 0 and !err
+
+getMemberId = (msg, member_name) ->
+  if (members[member_name.toLowerCase()]?)
+     id = members[member_name.toLowerCase()].id
+  msg.send "Unable to find person named: #{member_name}" unless id?
+  if id?
+     msg.send " * " + id
+
+addMember = (msg, card_id, member_name) ->
+  if (members[member_name.toLowerCase()]?)
+     id = members[member_name.toLowerCase()].id
+  msg.send "Unable to find person named: #{member_name}" unless id?
+  if id?
+     trello.put "/1/cards/#{card_id}/idMembers", {value: id}, (err, data) ->
+       msg.reply "Sorry captain, I couldn't add that member" if err
+       msg.reply "Success! #{member_name} was added" unless err
+
+getMemberCards = (msg, member_name) ->
+  if (members[member_name.toLowerCase()]?)
+     id = members[member_name.toLowerCase()].username
+  msg.send "Unable to find person named: #{user_name}" unless id?
+  if id?
+    trello.get "/1/search", {query: "@"+id, idBoards: board.id, modelTypes: "cards", card_fields: "name,shortLink,url"}, (err, data) ->
+      msg.reply "So sorry, I got an error and cannot give you that information" if err
+      msg.reply "I got the following cards for you" unless err
+      msg.send board.id
+      msg.send id
+      for cards in data.cards
+        msg.send " * " + cards.name+ " | " + cards.url
+        
+search = (msg, search) ->
+  if (search?)
+    msg.reply "I'm searching for results now"
+    trello.get "/1/search", {query: search, idBoards: board.id, modelTypes:"cards", card_fields: "name,shortLink,url"}, (err, data) ->
+      msg.reply "Sorry, I was unable to search at this time. Please try again later." if err
+      msg.reply "The following cards match your criteria" unless err
+      for cards in data.cards
+         msg.send " * " + cards.name + " | " + cards.url

```

Comment:
it might be better to convert this and the other places where you are concatenating strings into the string interpolation syntax for consistency.